site/developer-tools.html

```diff
@@ -793,6 +793,18 @@ <h4>In Spark unit tests</h4>
 The platform-specific paths to the profiler agents are listed in the 
 <a href="http://www.yourkit.com/docs/80/help/agent.jsp">YourKit documentation</a>.</p>
 
+<p><a name="local-cluster"></a></p>
+<h3>Local-cluster mode</h3>
+
+<p>When launching applications with spark-submit, besides options in 
+<a href="https://spark.apache.org/docs/latest/submitting-applications.html#master-urls">Master URLs</a>
+, set local-cluster option to emulate a distributed cluster in a single JVM.</p>
```

Comment:
shouldn't we explicitly say this is for unit testing only?

Reply:
I thought it is not necessary as it's on a developer-tools page but I'm totally fine with explicitly saying that here.
BTW if local-cluster mode ends up written in main documents (https://github.com/apache/spark/pull/33537), should it still be documented here?